base/src/org/spin/queue/notification/support/EMailSender.java

```diff
@@ -63,18 +65,31 @@ public void sendNotification(MADNotificationQueue notification) {
 		notification.getRecipients().forEach(recipient -> {
 			MClient client = MClient.get(notification.getCtx(), notification.getAD_Client_ID());
 			String eMailFrom = client.getRequestEMail();
+			String eMailPsw = client.getRequestUserPW();
+			int eMailConfigId = client.getAD_EMailConfig_ID();
+
+			if (notification.getAD_Org_ID() > 0) {
+				MOrg mOrg = MOrg.get(notification.getCtx(), notification.getAD_Org_ID());
+				MOrgInfo mOrgInfo = mOrg.getInfo();
+				if (mOrgInfo.getEMail() != null && mOrgInfo.getAD_EMailConfig_ID() > 0 && mOrgInfo.getRequestUserPW() != null) {
+					eMailFrom = mOrgInfo.getEMail();
+					eMailPsw = mOrgInfo.getRequestUserPW();
+					eMailConfigId = mOrgInfo.getAD_EMailConfig_ID();
+				}
+			}
+
 			MUser fromUser = null;
 			if(notification.getAD_User_ID() > 0) {
 				fromUser = MUser.get(notification.getCtx(), notification.getAD_User_ID());
 			}
 			//	Create instance
-			EMail email = new EMail(
-					client,
+			EMail email = new EMail(client,
+					eMailConfigId,
 					eMailFrom,
 					recipient.getAccountName(),
 					notification.getDescription(),
-					notification.getText()
-			);
+					notification.getText(),
+					false);
```

Comment:
@rcapecce If the content is HTML or not it should not be forced I understand that this should come from the notification and the notification should be based on the Mail Text Template

Reply:
In this line, we tried to respect what was already accepted before, so we tried to respect the same operation by incorporating the emailConfig method.
That is the reason why that parameter is burned, if you look at the signature of the method used previously, it is already burned from there.
https://github.com/adempiere/adempiere/blob/develop/base/src/org/compiere/util/EMail.java#L101
Currently the notifications do not see that they have support to determine if they are in html format or not.
In case you want to give support to this issue, how do you propose that the correction could be?